README.md

```diff
@@ -18,11 +18,23 @@ the use of the code as-is in production applications.
 
 ## Documentation
 
-### API documentation
+### Customization of the update process
+
+Mender offers several ways to customize the update process. You can find a
+high-level overview of the different approaches in the documentation section about the
```

Comment:
Suggestion:
in the `[Customization of the update process](https://docs.mender.io/overview/customize-the-update-process)` documentation.